index.html

```diff
@@ -51,15 +51,15 @@ <h3>
       <p>
         <br>
           This guide is the official style guide for Red&nbsp;Hat training and certification content, and for all other technical documentation except as stated.
-          Red&nbsp;Hat product documentation by Customer Content Services (CCS) follows the <a style="text-decoration: underline; font-size: 1em;" href="https://redhat-documentation.github.io/supplementary-style-guide/">Red&nbsp;Hat supplementary style guide</a> instead of this Red&nbsp;Hat Technical Writing Style Guide.
+          Red&nbsp;Hat product documentation follows the <a style="text-decoration: underline; font-size: 1em;" href="https://redhat-documentation.github.io/supplementary-style-guide/">Red&nbsp;Hat supplementary style guide</a> instead of this guide.
```

Comment:
```suggestion
          Red&nbsp;Hat product documentation follows the <a style="text-decoration: underline; font-size: 1em;" href="https://redhat-documentation.github.io/supplementary-style-guide/">Red&nbsp;Hat Supplementary Style Guide</a> instead of this guide.
```